### PR TITLE
installation: remove Python 2.7

### LINKED ISSUE #9: tests: remove python 2.7
Remove python 2.7 support to fix tests according to [the RFC](https://github.com/inveniosoftware/rfcs/blob/master/framework/0016-python2.md)

### LINKED ISSUE #11: release v1.0.2

### PR DESCRIPTION
**Pull request by diegodelemos**
* Closes #9.
* Closes #11

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,10 +19,14 @@ sudo: false
 language: python
 
 python:
-  - "2.7"
   - "3.5"
```
Comment (ppanero) [inline .travis.yml]: moderate: remove 3.5 too

File: setup.py [hunk 2]
```diff
@@ -79,8 +79,6 @@
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        "Programming Language :: Python :: 2",
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
```
Comment (ppanero) [inline setup.py]: moderate:
- remove 3.5
- add 3.7

File: .travis.yml (hunk 1 above)
Comment (diegodelemos) [reply .travis.yml]: Same as in https://github.com/inveniosoftware/idutils/pull/57, this needs clarification.

**Review by Glignos — APPROVED** [review]

**Comment by ppanero** [comment]
Build is passing https://travis-ci.org/github/inveniosoftware/base32-lib/builds/684296820 Not being reported back to GitHub. Known issue @fenekku Did you get any answer or that?

**Comment by fenekku** [comment]
@ppanero It looks all good from here (Travis passing and reporting on GitHub). This can be merged.

**Comment by ppanero** [comment]
@fenekku Cache.... sorry for the noise

**Pull request merged**